uberfire-project/uberfire-project-backend/src/main/java/org/guvnor/common/services/project/backend/server/POMServiceImpl.java

```diff
@@ -45,6 +55,9 @@
     private POMContentHandler pomContentHandler;
     private M2RepoService m2RepoService;
     private MetadataService metadataService;
+    private MavenXpp3Reader reader;
```

Comment:
I don't see this `reader` field used anywhere - do we need it?

Reply:
Was here due to the move of the code, but here is not needed, removed